src/services/shopify-client.ts

```diff
@@ -0,0 +1,25 @@
+export enum SubscriberListState {
+  ready,
+  synchronizing_contacts,
```

Comment:
What do you think about:

```suggestion
  synchronizingContacts,
```